### PR TITLE
RHDEVDOCS-5957: Content creation for GitOps 1.10.3 RN

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

This PR must be CP'ed to GitOps 1.12, GitOps 1.11, and GitOps 1.10 releases.

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-5957

Link to docs preview:

https://73166--docspreview.netlify.app/openshift-gitops/latest/release_notes/gitops-release-notes#release-notes-for-gitops-title-1-10-3_gitops-release-notes

SME review: @reginapizza
QE review: @varshab1210
Peer review: @Srivaralakshmi

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by varshab1210** [comment]
LGTM

**Review by reginaelyse — APPROVED** [review]
LGTM

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 Good work! Please add the errata updates section as discussed and put [WIP] in the PR description. Please keep the XX and NN parts in the links. Otherwise, the link will throw errors and PR builds won't pass. 

The rest of the content LGTM. Thank you!

**Comment by Dhruv-Soni11** [comment]
> @Dhruv-Soni11 Good work! Please add the errata updates section as discussed and put [WIP] in the PR description. Please keep the XX and NN parts in the links. Otherwise, the link will throw errors and PR builds won't pass.
> 
> The rest of the content LGTM. Thank you!

Hi @Srivaralakshmi - I have added the content for the Errata updates. Please check.

File: modules/gitops-release-notes-for-1-10-3.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+[id="release-notes-for-gitops-title-1-10-3_{context}"]
+= Release notes for {gitops-title} 1.10.3
+
+{gitops-title} 1.10.3 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-10-3_{context}"]
+== Errata updates
+
+[id="rhsa-2024-128924-02-gitops-1-10-3-security-update-advisory_{context}"]
+=== RHSA-xxxx:nnnn - {gitops-title} 1.10.3 security update advisory
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-for-1-10-3.adoc:16]: @Dhruv-Soni11 FYI and FYA:

NOTICE: [RHSA-2024:128924-02](https://errata.engineering.redhat.com/advisory/128924) has changed its advisory name to [RHSA-2024:1345-02](https://errata.engineering.redhat.com/advisory/128924)

Please update the PR and ping me, I will check and merge the RN and publish it. Thank you!

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 PTAL at my comment and let me know once the PR is ready. TY!

File: modules/gitops-release-notes-for-1-10-3.adoc (hunk 1 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-for-1-10-3.adoc]: Fixed it now @Srivaralakshmi. PTAL.

File: modules/gitops-release-notes-for-1-10-3.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+[id="release-notes-for-gitops-title-1-10-3_{context}"]
+= Release notes for {gitops-title} 1.10.3
+
+{gitops-title} 1.10.3 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-10-3_{context}"]
+== Errata updates
+
+[id="rhsa-2024:1345-02-gitops-1-10-3-security-update-advisory_{context}"]
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-for-1-10-3.adoc:15]:
```suggestion
[id="rhsa-2024:1345-gitops-1-10-3-security-update-advisory_{context}"]
```

File: modules/gitops-release-notes-for-1-10-3.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+=== RHSA-2024:1345-02 - {gitops-title} 1.10.3 security update advisory
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-for-1-10-3.adoc:16]:
```suggestion
=== RHSA-2024:1345 - {gitops-title} 1.10.3 security update advisory
```

File: modules/gitops-release-notes-for-1-10-3.adoc [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Issued: 2024-03-15
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024:1345-02[RHSA-2024:1345-02]
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-for-1-10-3.adoc:22]:
```suggestion
* link:https://access.redhat.com/errata/RHSA-2024:1345[RHSA-2024:1345]
```

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 PTAL at my comments and revise the PR.

File: modules/gitops-release-notes-for-1-10-3.adoc (hunk 2 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-for-1-10-3.adoc:15]: Fixed.

File: modules/gitops-release-notes-for-1-10-3.adoc (hunk 3 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-for-1-10-3.adoc:16]: Fixed.

File: modules/gitops-release-notes-for-1-10-3.adoc (hunk 4 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-for-1-10-3.adoc:22]: Fixed.

File: modules/gitops-release-notes-for-1-10-3.adoc [hunk 5]
```diff
@@ -0,0 +1,34 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
```
Comment (Srivaralakshmi) [inline modules/gitops-release-notes-for-1-10-3.adoc:6]: @Dhruv-Soni11 remove L6 blank line. Keep the module file metadata together. Refer to: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 Left a review comment. Please address it. TY!

File: modules/gitops-release-notes-for-1-10-3.adoc (hunk 5 above)
Comment (Dhruv-Soni11) [reply modules/gitops-release-notes-for-1-10-3.adoc]: Fixed this @Srivaralakshmi, please check and confirm. Thanks :)

**Pull request merged**